src/Reflow/Parser.hs

```diff
@@ -91,9 +91,12 @@ pgpBlockEnd = pack <$> string "[-- End signature information --]"
 pgpBlockContents :: Parser Text
 pgpBlockContents = pack <$> manyTill anyChar (lookAhead pgpBlockEnd)
 
+whiteSpaceOnlyLine :: Parser String
+whiteSpaceOnlyLine = do
```

Comment:
Not to beat the "applicative parser" drum too much, but I think this could be:

```haskell
whiteSpaceOnlyLine = many (char ' ') <* endOfLine
```

(Also, personally I'd lowercase the S in `whiteSpace`.)